docs/source/architecture/period_duration.rst

```diff
@@ -0,0 +1,61 @@
+Period Duration
+===============
+
+In the NuCypher Network, a *Period* is the minimum unit for policy duration. This applies to sharing policy durations
+and also corresponds to the frequency that Workers must make an on-chain commitment to being online and available.
+Periods begin at UTC midnight.
+
+Genesis
+-------
+
+At the Network launch on October 15th, 2020, the duration of a period was 24 hours. As a result, before UTC midnight,
+Workers needed to make a per period (daily) on-chain commitment transactions indicating availability for the following
+period (after UTC midnight).
+
+The cost of the commitment transaction was approximately 200k gas but could cost more depending on the number of
+sub-stakes being staked. Given the significant increase in Ethereum gas since launch, the cost of the daily
```

Comment:
```suggestion
The cost of the commitment transaction is approximately 200k gas units but may be higher depending on the number of
sub-stakes a staker has. Given the significant increase in Ethereum gas price since network launch, the cost of the daily
 ```

Reply:
✔️